highcharts-behavior.html

```diff
@@ -35,6 +35,7 @@
             yZoom: {type: Boolean, value: false, reflectToAttribute: true, observer: "_zoomUpdate"},
             loading: {type: Boolean, value: false, observer: "_loadingUpdate", reflectToAttribute:true},
             loadingMessage: {type: String, value: ""},
+            renderer: {type: Object, computed: "_renderer(_chart)"},
```

Comment:
I just tried this update but it throw an error: Cannot read property 'image' of undefined.
And yes I didn't notice the mutation during the init phase so, may I suggest this update instead ?

`renderer: {type: Object, computed: "_renderer(_chart.renderer)"},`

Then the mutation will be triggered only when "_chart.renderer" change (according to Polymer specs for mutation) : https://www.polymer-project.org/1.0/docs/devguide/observers#define-a-computed-property

Seems good ?